### ISSUE TITLE
Using a two way picker on Media to Media doesn't work..

### ISSUE THREAD

**Issue by Rizen-Kris** [scyllagroup/TwoWayPicker#2]
The option to pick document type property of the editor is an empty drop down even when the other media type has a picker.

**Comment by markadrake** [comment]
Hi @Rizen-Kris – I'm going to open this as a potential bug in JIRA. We'll update you - may even have questions for you. If you have any more details or explicit instructions to reproduce please leave them here and I'll share with the team!

Thanks!

**Comment by jmcaveney** [comment]
I don't think we ever planned for it to work with media. We'll be addressing this package at some point in the first part of next year and will have to add this as a potential feature.
